shared/src/gucs.rs

```diff
@@ -64,6 +69,15 @@ impl PostgresGlobalGucSettings {
             GucFlags::default(),
         );
 
+        GucRegistry::define_bool_guc(
+            "paradedb.enable_custom_scan",
+            "Enable ParadeDB's custom scan",
+            "Enable ParadeDB's custom scan",
+            &self.enable_custom_scan,
+            GucContext::Userset,
+            GucFlags::default(),
+        );
```

Comment:
we should document this.  And should note that if our custom scan is disabled, then scores and snippets WILL NOT WORK.